app/views/bookings/show.html.erb

```diff
@@ -10,7 +10,6 @@
     <p><%= @booking.completed %></p>
   </div>
 
-<%= link_to "Review your job", new_job_booking_review_path, class: "btn btn-outline-secondary mb-2" %>
 <%= link_to "Back to bookings", bookings_path(@bookings), method:'get', class: "btn btn-outline-secondary mb-2" %>
```

Comment:
I don't think we need the `method: get` part here since GET is the default method.